api/src/main/java/com/sap/xsa/security/container/XSUserInfo.java

```diff
@@ -192,15 +249,19 @@
 	 * @throws XSUserInfoException
 	 *             if attribute is not available in the authentication token
 	 */
+	// TODO 14.01.20 c5295400: cannot be replaced yet
```

Comment:
Maybe we can explain it according to this code snippet:
https://github.com/SAP/cloud-security-xsuaa-integration/blob/master/spring-xsuaa/src/main/java/com/sap/cloud/security/xsuaa/token/XsuaaToken.java#L201